platform-coop-toolkit/coop-map-directory-index
# feat: add individual profile, clean up account/login/registration templates

## Description

This PR implements the full profile for individuals. Profile editing will be addressed in a later PR. It also applies full Pinecone formatting to account management templates: password management, email address management, password resets, log in, log out, etc.

## Steps to test

1. Check out branch.
2. Update dependencies and build: `npm ci && npm run build:production`
3. Navigate to an individual profile, and review.
4. Log in and out, change password, manage email addresses.

**Expected behavior:** Matches designs.

## Additional information

I have not modified default spelling of `email` (`e-mail`) in account management template form fields and placeholders. This would be nice to do but involves some effort.

## Related issues

- Resolves #7.
- Resolves #15.
